### PR TITLE
Add code to pull global credential & set envvars for Opsgenie heartbeats

### PR DESCRIPTION
**Pull request by doctoryes**
Another try to enable Opsgenie heartbeat detection of stuck Jenkins jobs - here's the heartbeats:

https://edx.app.opsgenie.com/settings/heartbeat

The new global credential named "opsgenie_heartbeat_config_key" can be seen here:

http://jenkins.analytics.edx.org/credentials/

This PR pulls that global credential into all relevant jobs, along with three public environment variables, so that the OpsGenie API can be successfully called to enable/ping/disable each heartbeat.

### CODE REVIEW

**Comment by doctoryes** [comment]
The previous attempt to enable this functionality was reverted here:
https://github.com/edx/jenkins-job-dsl/pull/1170
The technique used in that PR leaked the API key to the Jenkins log - this technique should *not*.

**Review by bmedx — APPROVED** [review]
LGTM

File: dataeng/jobs/analytics/Enrollment.groovy [hunk 1]
```diff
@@ -31,6 +27,26 @@ class Enrollment {
...
                 }
+                parameters common_parameters(allVars, env_config)
+                parameters from_date_interval_parameter(allVars)
+                parameters to_date_interval_parameter(allVars)
+                parameters {
+                    stringParam('BASE_OUTPUT_URL', env_config.get('BASE_OUTPUT_URL', allVars.get('BASE_OUTPUT_URL')), '')
+                    stringParam('OUTPUT_DIR', env_config.get('OUTPUT_DIR', allVars.get('OUTPUT_DIR')), '')
+                    stringParam('OUTPUT_SUFFIX', env_config.get('OUTPUT_SUFFIX', allVars.get('OUTPUT_SUFFIX')), '')
+                    stringParam('CREDENTIALS', env_config.get('CREDENTIALS', allVars.get('CREDENTIALS')), '')
+                }
```
Comment (pwnage101) [inline dataeng/jobs/analytics/Enrollment.groovy:38]: What's up with all these new parameters?  Seems unrelated to heartbeats?

**Review by pwnage101 — COMMENTED** [review]
One question below, and also just wanted to make sure you'll codify the new credential you added via ansible, e.g. https://github.com/edx-ops/analytics-secure/pull/446

File: dataeng/jobs/analytics/Enrollment.groovy (hunk 1 above)
Comment (doctoryes) [reply dataeng/jobs/analytics/Enrollment.groovy:38]: Ugh - smh. This is copy pasta from another job. I'll remove it.

Comment (doctoryes) [reply dataeng/jobs/analytics/Enrollment.groovy:38]: Nice catch!

Comment (doctoryes) [reply dataeng/jobs/analytics/Enrollment.groovy]: Removed now.
